lms/static/scripts/frontend_apps/index.tsx

```diff
@@ -69,6 +70,13 @@ export function init() {
     }
   }
 
+  if (config.emailNotifications) {
+    const emailPreferencesService = new EmailPreferencesService(
+      window.fetch.bind(window)
```

Comment:
Aha. That makes me wonder if just `fetch` would also work.

Reply:
> Aha. That makes me wonder if just `fetch` would also work.

Nah, it has to be bound.

Reply:
Passing just `fetch` doesn't work because the class will call `this._fetch` internally which will set the receiver to `this` (ie. the `EmailPreferencesService` class instance) if `bind` hasn't been used.